### PR TITLE
adding SSWG meeting notes

### PR DESCRIPTION
**Pull request by masters3d**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 `BaggageContext` is a minimal (zero-dependency) "context" library meant to "carry" baggage (metadata) for cross-cutting tools such as tracers.
 It is purposefully not tied to any specific use-case (in the spirit of the [Tracing Plane paper](https://cs.brown.edu/~jcmace/papers/mace18universal.pdf)'s BaggageContext), however it should enable a vast majority of use cases cross-cutting tools need to support. Unlike mentioned in the paper, our `BaggageContext` does not implement its own serialization scheme (today).
 
-See https://github.com/slashmo/gsoc-swift-tracing for actual instrument types and implementations which can be used to deploy various cross-cutting instruments all reusing the same baggage type.
+See https://github.com/slashmo/gsoc-swift-tracing for actual instrument types and implementations which can be used to deploy various cross-cutting instruments all reusing the same baggage type. See [SSWG meeting notes](https://gist.github.com/ktoso/4d160232407e4d5835b5ba700c73de37#swift-baggage-context--distributed-tracing) for more information.
```
Comment (slashmo) [inline README.md]:
```suggestion
See https://github.com/slashmo/gsoc-swift-tracing for actual instrument types and implementations which can be used to deploy various cross-cutting instruments all reusing the same baggage type. More information can be found in the [SSWG meeting notes](https://gist.github.com/ktoso/4d160232407e4d5835b5ba700c73de37#swift-baggage-context--distributed-tracing).
```

**Review by slashmo — CHANGES_REQUESTED** [review]
LGTM, thanks 👍 I'd like to avoid starting both sentences with "See" so I phrased it a bit differently.

File: README.md (hunk 1 above)
Comment (ktoso) [reply README.md]: Either spelling is okey 🙂

**Review by ktoso — APPROVED** [review]
LGTM, merge at will with or without the rewording @slashmo.

Thanks for the additional link @masters3d, makes sense.

**Review by slashmo — APPROVED** [review]
Thanks again 👌

**Pull request merged**